File: snippets/php/update_a_subaccount_s_quota.php
require_once('/path/to/mailchimp-transactional-php/vendor/autoload.php');
function run($subAccount)
{
    try {
        $mailchimp = new MailchimpTransactional\ApiClient();
        $mailchimp->setApiKey("YOUR_API_KEY");
        $response = $mailchimp->subaccounts->update($subAccount);
        if (isset($response->custom_quota)) {
            echo "Success! This account's new sending quota is " . $response->custom_quota;
        } else {
            echo "Could not update subaccount: <br>";
            print_r($response);
        }
    } catch (Error $e) {
        echo 'Error: ', $e->getMessage(), "\n";
    }
}

$subAccount = [
    "id" => "UNIQUE_ACCOUNT_ID",
    "custom_quota" => 100
];
run($subAccount);

File: snippets/ruby/update_a_subaccount_s_quota.rb
require 'MailchimpTransactional'

client = MailchimpTransactional::Client.new(ENV['YOUR_API_KEY'])
account_id = "UNIQUE_ACCOUNT_ID"

fields = {
  # account id that you want updated
  id: account_id,
  # fields you want updated
  custom_quota: 100
}

begin
  account = client.subaccounts.update(fields)
  puts account
rescue MailchimpTransactional::ApiError => e
  puts "Error: #{e}"
end